packages/lib/src/types/options.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  GetHighlightedElementsFunction,
+  SankeyLink,
+  SankeyLinkProps,
+  SankeyNodeProps,
+} from './alluvial';
+import { ColorPalette } from './dataset';
+import { SankeyNode } from 'd3-sankey';
```

Comment:
Some of these could also use the `type` prefix on the `import`

Reply:
Aah indeed!
Missed that - thanks will update :)